cif_ed.dic

```diff
@@ -0,0 +1,643 @@
+    that technique.
+;
+
+save_ED_GROUP
+
+    _definition.id               ED_GROUP
+    _definition.scope            Category
+    _definition.class            Head
+    _definition.update           2024-04-22
+    _description.text
+;
+     The ED_GROUP data items describe atomic information specific to
+     crystallographic structure determination using electron
+     diffraction.
+;
+    _name.category_id            CIF_ED
+    _name.object_id              ED_GROUP
+
+    _import.get
+        [{'dupl':Ignore  'file':cif_core.dic  'mode':Full  'save':CIF_CORE}]
+
+save_
+
+save_chemical.absolute_configuration
+
+    _definition.id                '_chemical.absolute_configuration'
+    _alias.definition_id          '_chemical_absolute_configuration'
+    _definition.update            2024-04-22
+    _description.text
+;
+    Necessary conditions for this assignment are given by
+         Flack, H. D. & Bernardinelli, G. (1999). Acta Cryst. A55, 908-915.
+                https://doi.org/10.1107/S0108767399004262
+                https://www.iucr.org/paper?sh0129
+         Flack, H. D. & Bernardinelli, G. (2000). J. Appl. Cryst. 33, 1143-1148.
+                https://doi.org/10.1107/S0021889800007184
+                https://www.iucr.org/paper?ks0021
+;
+    _name.category_id             chemical
+    _name.object_id               absolute_configuration
+    _type.purpose                 State
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Text
+
+    loop_
+      _enumeration_set.state
+      _enumeration_set.detail
+         rm
+;
+         'reference molecule' Absolute configuration established by the
+         structure determination of a compound containing a chiral reference
+         molecule of known absolute configuration.
+;
+         ad
+;
+         'anomalous dispersion' Absolute configuration established by a-d
+         effects in diffraction measurements on the crystal.
+;
+         dyn
+;
+         Absolute configuration determined directly from dynamical refinement.
+;
+         rmad
+;
+         'rm + ad' Absolute configuration established by the structure
+         determination of a compound containing a chiral reference molecule of
+         known absolute configuration and confirmed by a-d effects in
+         diffraction measurements on the crystal.
+;
+         syn
+;
+         'synthetic' Absolute configuration has not been established by
+         anomalous-dispersion effects in diffraction measurements on the
+         crystal. The enantiomer has been assigned by reference to an
+         unchanging chiral centre in the synthetic procedure.
+;
+         unk
+;
+         'unknown' No firm chemical or a-d evidence for an assignment is
+         available. An arbitrary choice of enantiomer has been made.
+;
+         .
+;
+         Inapplicable.
+;
+
+save_
+
+save_diffrn.precession_angle
```

Comment:
Similar question to the one about `_diffrn_measurement.tracking` and `_diffrn_measurement.tracking_method`. The `_diffrn.precession_angle` and `_diffrn_measurement.method_precession` items seems to be quite tightly connected. This may lead to some data anomalies like having a dataset with `_diffrn_measurement.method_precession` set to `no`, but the `_diffrn.precession_angle` still having a value. Maybe having only the `_diffrn.precession_angle` suffice? That is, the presence of this item would indicate "yes"?

Or are there some situations where the exact angle is not known and thus cannot be provided?

Reply:
Will include as an issue for community consideration.